### ISSUE TITLE
CentOS current packages

### ISSUE THREAD

**Issue by deajan** [sahib/rmlint#186]
Wonderful tool you have built.
I've used it on a zfs filesystem with 1.2M files in 3TB. It was surprisingly fast, took only about 3 hours to find 40% if dupes.

But I had trouble finding an rpm for CentOS 7. The copr repo doesn't provide anything for lower than Fedora 21. It's a shame such a tool isn't available on the main Redhat platform.
I've ended up using the opensuse RPM built service, which had rmlint 2.2.0.

Any chances to see current builds for CentOS 7, in EPEL perhaps ?

**Comment by sahib** [comment]
Hello,

we've decided to only create and maintain packages for platforms that we're actively using. Everything else is usually a lot of work when releasing new versions, especially when you want to get it into the main repository of a distribution. See also issue #143 for more information. We're relying here on the help of the community to create and maintain packages. 

In the case of copr: I don't actively maintain that anymore, since I moved backed to Arch since some time. Maybe you'd like to setup an own, more current repository? It's not that hard, since we already have a ``.spec`` file to build a rpm in ``pkg/fedora/rmlint.spec``. I'd be glad to help on setting it up.
